ord_schema/validations.py

```diff
@@ -138,11 +138,36 @@ def ensure_details_specified_if_type_custom(message):
     return message
 
 
+def reaction_has_internal_standard(message):
+    return any(compound.reaction_role ==
```

Comment:
Would you mind splitting this up across multiple lines? Multiple nested inline for loops is pretty hard to read.

There are a few other instances later on as well.